app/src/main/java/com/example/playlistmaker/SearchActivity.kt

```diff
@@ -4,21 +4,48 @@ import android.os.Bundle
 import android.text.Editable
 import android.text.TextWatcher
 import android.view.View
+import android.view.inputmethod.EditorInfo
 import android.view.inputmethod.InputMethodManager
 import android.widget.EditText
 import android.widget.ImageView
 import android.widget.LinearLayout
+import android.widget.TextView
+import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
 import kotlin.random.Random
 
 class SearchActivity : AppCompatActivity() {
 
     private val trackList: MutableList<Track> = mutableListOf()
 
+    private val trackAdapter = TrackAdapter(trackList)
+
+    private lateinit var backBtn: com.google.android.material.appbar.MaterialToolbar
```

Comment:
```suggestion
    private lateinit var backBtn: MaterialToolbar
```
А все что com.google... можно вынести в импорт
![image](https://github.com/DoNatPanic/playlist-maker/assets/129396632/04bc0248-f35e-48ca-ae4e-27e49a2510f3)